.github/workflows/bump-cli.yml

```diff
@@ -23,10 +36,18 @@ jobs:
         with:
           fetch-depth: 1
       - name: Bump CLI
+        if: ${{ inputs.option == 'replace' }}
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           scripts/replace-cli-version.sh
+      - name: Prepend another version
+        if: ${{ inputs.option == 'prepend' }}
+        run: |
+          cat supported_cli_versions.json | jq '. |= ["${{ inputs.version }}"] + .' > supported_cli_versions_temp.json
+          mv supported_cli_versions_temp.json supported_cli_versions.json
+          echo "LATEST_VERSION=${{ inputs.version }}" >> $GITHUB_ENV
+          echo "PREVIOUS_VERSION=`jq -r '.[1]' supported_cli_versions.json`" >> $GITHUB_ENV
```

Comment:
Let me know if I should move this into a separate script similar to the `scripts/replace-cli-versions.sh` 😄